src/layers/models/gpkgValidator.ts

```diff
@@ -0,0 +1,119 @@
+import { extname } from 'node:path';
+import { Logger } from '@map-colonies/js-logger';
+import { BadRequestError } from '@map-colonies/error-types';
+import { inject, injectable } from 'tsyringe';
+import { SERVICES } from '../../common/constants';
+import { IConfig } from '../../common/interfaces';
+import { PixelRange } from '../interfaces';
+import { SQLiteClient } from '../../serviceClients/sqliteClient';
+import { Grid } from '../interfaces';
+
+@injectable()
+export class GpkgValidator {
```

Comment:
Please rename to "GpkgUtilities" and add the following methods:
* isGpkg(): boolean // in order to decide if GPKG validation is needed
* GetGrid():  Grid | undefined // for layersManager usage when creating merge task